job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -13,8 +13,8 @@ def NEXT_PRODUCT_BRANCH='main'
 def NEXT_RHBA_VERSION_PREFIX=Constants.RHBA_VERSION_PREFIX
 
 def CURRENT_PRODUCT_VERSION=Constants.CURRENT_PROD_VERSION
```

Comment:
is https://github.com/kiegroup/kie-jenkins-scripts/blob/ba40179a95eba368e3c0666c919893b2ce90ceef/job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy#L41 value ok? shouldn't it be `7.13.0`? and what about `NEXT_PROD_VERSION`? :thinking:

Reply:
https://github.com/kiegroup/kie-jenkins-scripts/pull/1226/files